examples/complete/main.tf

```diff
@@ -69,7 +70,7 @@ module "event_streams" {
         attributes = [
           {
             "name" : "endpointType",
-            "value" : "private"
+            "value" : "public"
```

Comment:
Not sure we want to include a CBR rule that mandates using public endpoint. Thats not a secure and good practise. Is there a reason for this change? If so, perhaps we only keep CBRs in the fscloud example?

Reply:
Upgrade tests runs on this complete example which cannot run in GHA environment due to private endpoint and the test is failing almost everytime.